docs/guide/http_cache.md

```diff
@@ -318,6 +318,10 @@ fastcgi_param SYMFONY_HTTP_CACHE 0;
 fastcgi_param SYMFONY_TRUSTED_PROXIES "193.22.44.22";
 ```
 
+!!! caution "Trusted proxies when using SSL offloader / loadbalancer"
+
+    If your installation works behind Varnish and SSL offloader (like haproxy) then you need   to add `127.0.0.1` address to the `SYMFONY_TRUSTED_PROXIES`. Otherwise, you might notice incorrect schema (`http` instead of `https`) in the URLs for the images or other binary files.
```

Comment:
`need   to ` => `need to `

suggestion: `other binary files when they are rendered inline Symfony _(as used by file based field templates)_, as opposed to via ESI.`